tests/functional/UseCase/Table/DeleteTableRowsTest.php

```diff
@@ -282,6 +284,9 @@ private function deleteRows(
         if (array_key_exists('whereFilters', $commandInput)) {
             $command->setWhereFilters($commandInput['whereFilters']);
         }
+        if (array_key_exists('whereRefTableFilters', $commandInput)) {
```

Comment:
trochu jsem doufal, ze to cele mergnes do novych whereFilters, ale to bylo uz dano tim common-driverem. Nevadi